examples/echo/main.c

```diff
@@ -33,6 +33,62 @@ g_date_time_format_iso8601 (GDateTime *datetime)
 }
 #endif
 
```

Comment:
Just note to commit log of `e690d5e677eb2986f0edb73a844b5a173e247f42` the commit message is on one very long line. I would add there some line breaks.